### ISSUE TITLE
Why does the output of graph matching have two loss terms, one positive and one negative, so will they be effective if they are added together to make the loss?

### ISSUE THREAD

**Issue by lhc11171552** [hh23333/PVPM#15]

**Comment by hh23333** [comment]
@lhc11171552 
Hi,
The positive term is the binary classification loss for training the PVP module to predict the right visibility score of each part. And the negative term aims to maximize the object function of graph matching. The final loss function has already been the sum of them.
